README.md

```diff
@@ -54,6 +54,15 @@ withPipeline(type, product, component) {
 }
 ```
 
+The opinionated pipeline uses the following branch mapping to deploy applications to different environments.
+
+Branch | Environment
+--- | ---
+`master` | `aat` then `prod`
+`demo` | `demo`
+`perftest` | `perftest`
+PR branch| `preview` (ASE or AKS depending on your config)
```

Comment:
does this deploy or run plan only?
preview doesn't work very well for shared infra because it gets deleted constantly, for teams like bulk-scanning it's a pain in the butt because they need devops intervention 

what was being done instead in their pipeline is running plan against AAT...